Dockerfile

```diff
@@ -38,7 +38,7 @@ ARG NEXUS_REPOSITORY_MANAGER_COOKBOOK_URL="https://github.com/sonatype/chef-nexu
 ADD solo.json.erb /var/chef/solo.json.erb
 
 # Install using chef-solo
-RUN curl -L https://www.getchef.com/chef/install.sh | bash \
+RUN curl -L https://www.getchef.com/chef/install.sh | bash -s -- -v 14.12.9 \
```

Comment:
Suggestion: add a note/comment above here pointing to this PR and some details on why we locked into the version.